lexicons/app/bsky/feed/defs.json

```diff
@@ -109,7 +116,7 @@
             "refs": ["#threadViewPost", "#notFoundPost", "#blockedPost"]
           }
         },
-        "isLikedByRootAuthor": { "type": "boolean" }
+        "rootAuthor": { "type": "ref", "ref": "#rootAuthorState" }
```

Comment:
I have a small issue with the name, which is that "author" fields are typically profile views, as in the case of `replyRef.grandparentAuthor` or `postView.author`.

What do you think of reframing this field as "thread-specific state" for the post.  Other state that could end up there would be various post rankings within a thread, whether the post is highlighted within a thread, etc.  In that case it might look like:
```json5
{
  "post": {/*...*/},
  "parent": {/*...*/},
  "replies": [{/*...*/}],
  "threadContext": {
    "like": "at://..."
  }
}
```

If we need another set of eyes or a tie-breaker on this, Daniel has opinions and good sensibilities around this kind of thing too.

Reply:
I like the reframing to `threadContext`. mainly because i think it's likely that we'll continue to add additional pieces of threadContext

I also like the further clarification of `like`. `opLike` is a little to jargony I think, and prefer to use something with `root` in it. I like `rootAuthorLike` 👍

Reply:
Sweet I dig that, let's do that then if Rafael is also down with it:
```json5
{
  "post": {/*...*/},
  "parent": {/*...*/},
  "replies": [{/*...*/}],
  "threadContext": {
    "rootAuthorLike": "at://..."
  }
}
```